images.go

```diff
@@ -595,20 +594,22 @@ func (r *imageStore) BigDataSize(id, key string) (int64, error) {
 		return size, nil
 	}
 	if data, err := r.BigData(id, key); err == nil && data != nil {
-		if r.SetBigData(id, key, data) == nil {
+		if err = r.SetBigData(id, key, data, digestManifest); err == nil {
```

Comment:
Forcing the caller to supply `digestManifest` here, because of an ultimately very internal implementation decision, is not great.

Couldn’t this just do
```go
image.BigDataSizes[key] = int64(len(data))
r.Save()
```
or something like that (possibly with a helper shared with `SetBigData`?